### DISCUSSION TITLE
Great project! And some suggestions

### DISCUSSION THREAD

**Discussion by carlopi** [jameslittle230/stork#150] [Ideas]
Congrats on this great project!

I am unable to draw, but a logo inspired by [this](https://www.google.com/search?q=stork+flying+from+below&tbm=isch&ved=2ahUKEwiUnO7su-7uAhUFG-wKHVw5BU4Q2-cCegQIABAA&oq=stork+flying+from+below&gs_lcp=CgNpbWcQAzoCCAA6BAgAEB46BggAEAUQHlCjtwJYrMUCYOXGAmgAcAB4AIABO4gBqwSSAQIxMZgBAKABAaoBC2d3cy13aXotaW1nwAEB&sclient=img&ei=1MIrYJTiEIW2sAfc8pTwBA&bih=912&biw=1848&client=ubuntu&hs=9GO&safe=active#imgrc=mhchvNNKKW9AjM) would be cool.
The idea would be something similar to airplane are drawn from above, tilted 45 degrees to fit in an icon (similar to :airplane:, check also here: https://emojipedia.org/airplane/) 

I have some not-yet formed ideas about how to organize some kind of hierarchical search (sort of a trie, where inner data is fetched only whenever needed). I will get back when I have a more proper suggestion.

On the nitpick side: it's either WebAssembly or Wasm (never WASM!). And probably also JavaScript instead of Javascript.

Great work, keep up!

**Comment by jameslittle230** [comment]
Thank you for the ideas!
